src/LuaTransformer.ts

```diff
@@ -1,7 +1,7 @@
 import * as path from "path";
 import * as ts from "typescript";
 import { CompilerOptions, LuaTarget } from "./CompilerOptions";
-import { DecoratorKind } from "./Decorator";
+import { DecoratorKind, Decorator } from "./Decorator";
```

Comment:
Looks like an accidental import. (We should probably rename this to Directives or something soon to prevent confusion)